encoder_node:
  ros__parameters:
    i2c_bus: 1
    home_offset: 3030   # ปรับให้ 0° ตรงจริง

controller_node:
  ros__parameters:
    step_pin: 14
    dir_pin: 15
    enable_pin: 18

    base_steps_per_rev: 100
    microstepping: 4
    direction_cw: 1
    direction_ccw: 0

    offset_deg: -10.0
    max_velocity_deg_per_sec: 200.0
    min_velocity_deg_per_sec: 20.0
    tolerance_deg: 0.5
    deceleration_zone_deg: 10.0

    default_speed_deg_per_sec: 150.0
    stop_on_crossing: false